src/pages/ContactUs/index.js

```diff
@@ -0,0 +1,259 @@
+    color: "white",
+  },
+  messageBox: {
+    marginTop: "30px",
+    marginLeft: "30px",
+    marginRight: "30px",
+  },
+  notchedOutline: {
+    borderWidth: "2px",
+    borderColor: "white !important",
+    borderRadius: "25px",
+  },
+  button: {
+    padding: "'0.6rem','1.3rem'",
+    backgroundColor: "#fff",
+    fontSize: "0.95rem",
+    color: "#1abc9c",
+    lineHeight: "1",
+    borderRadius: "25px",
+    transition: "0.3s",
+    marginTop: "25px",
+    marginLeft: "70px",
+    marginRight: "70px",
+    marginBottom: "30px",
+    "&:hover": {
+      backgroundColor: "transparent",
+      color: "#fff",
+    },
+  },
+}));
+
+function ContactUs(props) {
+  const classes = useStyles();
+  const history = useHistory();
+
+  return (
+    <>
+      <AppBar position="static" className={classes.appBar}>
+        <Toolbar variant="dense">
+          <IconButton
+            edge="start"
+            className={classes.backButton}
+            color="inherit"
+            onClick={() => history.goBack()}
+          >
+            <ArrowBackIcon />
+          </IconButton>
+          <Typography variant="h6" color="inherit" className={classes.head}>
+            Contact Us
+          </Typography>
+          <IconButton edge="end" className={classes.menuButton} color="inherit">
+            <MoreVertIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+      <div>
+        <Typography className={classes.title} variant="h2" color="secondary">
+          Let&apos;s get in touch
+        </Typography>
+        <Typography className={classes.textHead} variant="body2">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe dolorum adipisci recusandae
+          praesentium dicta!
+        </Typography>
+        <Grid container>
+          <div className={classes.info}>
+            <div className={classes.information}>
+              <LocationCityIcon className={classes.icons} />
+              <Typography>92 Cherry Drive Uniondale, NY 11553</Typography>
+            </div>
+            <div className={classes.information}>
+              <EmailIcon className={classes.icons} />
+              <Typography>lorem@ipsum.com</Typography>
+            </div>
+            <div className={classes.information}>
+              <PhoneInTalkIcon className={classes.icons} alt="" />
+              <Typography>123-456-789</Typography>
+            </div>
+          </div>
+        </Grid>
+        <Typography className={classes.title} variant="h2" color="secondary">
+          Connect with us :
+        </Typography>
+        <div className={classes.socialIcons}>
+          <FacebookIcon href="#" className={classes.socialIcons} />
+          <TwitterIcon href="#" className={classes.socialIcons} />
+          <InstagramIcon href="#" className={classes.socialIcons} />
+          <LinkedInIcon href="#" className={classes.socialIcons} />
+          <YouTubeIcon href="#" className={classes.socialIcons} />
```

Comment:
Give codeforcause links here,

You can found all in codeforcause.org footer

Reply:
Ok, @Abhishek-kumar09 I will do all the requested changes :) 
I am also implementing the changes suggested by @sakshi-choudhary.

Thanks for reviewing my PR 👍